src/interfaces/utils-interfaces/form-type.ts

```diff
@@ -3,4 +3,5 @@ import Vue from 'vue'
 export interface FormType extends HTMLFormElement {
   reset(): void
   validate(): boolean
+  resetValidation(): void
```

Comment:
I can't find the complete form API at the moment. Are you sure this is a valid method?

Reply:
It is indeed. 
If we use `reset()` it will reset the input values as well. 

https://vuetifyjs.com/en/api/v-form/#functions

Reply:
OK, that's perfect for our "high level validation" use. Good find!

It if were me, I'd add a link to the v-form API as a comment on this IF :)

Reply:
Great idea, will do!